src/main/webapp/resources/js/pages/announcement/components/ViewAnnouncement.jsx

```diff
@@ -42,13 +41,16 @@ function ViewUnreadAnnouncementModal({
       onOk() {
```

Comment:
i18n on the okText property of this modal.

Reply:
Please see 3f3d1f15d39f3b619ddd2efe57e020a0a6717174.